### PR TITLE
Core alert fields aliases _ part 3 (sasha)

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added aliases - part 3

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json [hunk 1]
```diff
@@ -28,5 +28,39 @@
...
+        {
+          "cliName": "qualysfimeventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "solarwindseventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "taniumtreatresponsescanconfigid",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json:62]: Is this related?

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Host_FQDN.json [hunk 2]
```diff
@@ -28,5 +28,19 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Host_FQDN.json:30]: Need to validate if this can be multiselect

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json [hunk 3]
```diff
@@ -39,11 +39,10 @@
           "cliName": "carbonblackesprocessname",
           "type": "shortText"
         },
-       
+        
         {
-            "cliName": "digitalguardianprocessname",
-            "type": "shortText"
+            "cliName": "digitalguardianparentprocessname",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json:44]: Initiate by is the process name not parent process name
Should be aliased to CGO name (causality_actor_process_image_name)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json (hunk 1 above)
Comment (ssokolovich) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json:62]: image
Followed this record. Seems like I should take the one from above right? @David-BMS

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Host_FQDN.json (hunk 2 above)
Comment (ssokolovich) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Host_FQDN.json:30]: @David-BMS - pls update once you have an answer

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json (hunk 3 above)
Comment (ssokolovich) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json:44]: Sure - changed the x2_filed map + changed the alias to the  `digitalguardianprocessname`

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json (hunk 1 above)
Comment (David-BMS) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json:62]: Yep

Comment (ssokolovich) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json:62]: Done

**Review by David-BMS — APPROVED** [review]
